docs/proposals/kubelet-eviction.md

```diff
@@ -76,8 +76,10 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
 | nodefs.available   | nodefs.available := node.stats.fs.available |
 | nodefs.inodesFree | nodefs.inodesFree := node.fs.inodesFree |
+| nodefs.inodesPercentAvailable | nodefs.inodesPercentAvailable := node.fs.inodesFree / node.fs.inodes |
```

Comment:
This basically requires us to change the syntax for eviction thresholds.

`<eviction-signal><operator><quantity | int%>`

internally in-code, we would need to treat literal values and percentage values as different underlying signals because when we make our observations on the state of the node, we would need to make both the literal and percentage based observation as separate tokens.  not all signals in future may support the ability to express a percentage, so we then need to write special case code to say this particular signal supports one or both syntax choices.

see: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/eviction/helpers.go#L429

As a user, I worry that failure to omit a `%` could cause drastically different results that only manifest themselves when bad things are happening to their nodes.  I said to evict when `nodefs.inodesFree<10`, oh wait, I meant 10%!

I think we should just be consistent with how things like HPA work and use:

* `inodesFree` - literal value
* `inodesFreePercentage` - percentage values only
* `nodefs.available` - literal value
* `nodefs.availablePercentage` - percentage values only

that said, why don't we just use `inodesAvailable` as well?